src/main/resources/static/api.yml

```diff
@@ -3847,6 +3850,71 @@ components:
             - schemaVersion
             - outdatedSchema
             - status
+    ApplicationSummary:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        person:
+          $ref: '#/components/schemas/Person'
+        createdAt:
+          type: string
+          format: date-time
+        submittedAt:
+          type: string
+          format: date-time
+      discriminator:
+        propertyName: type
```

Comment:
Where does this `propertyName` get sent in the API? I'm trying to build the frontend changes now and I get a 400 when trying to send as `UpdateApprovedPremisesApplication` object